### PR TITLE
[Waiting for 2.4] fixed timestamp sorting on ansible history

### PR DESCRIPTION
**Pull request by ChunxiAlexLuo**
Signed-off-by: Chunxi Luo chuluo@redhat.com

https://github.com/open-cluster-management/backlog/issues/13805

before 

Screen Shot 2021-07-15 at 10 26 05 AM

after

Screen Shot 2021-07-15 at 3 11 30 PM

### CODE REVIEW

**Comment by dhaiducek** [comment]
Holy moly! Removing PatternFlyTable is 2,400 lines of code??? RIP 🤣 

This looks good to me, but since it's deferred for `2.4`, we should wait until that's open to merge it so CICD isnt' caught off guard.

/lgtm
/hold

**Review by dhaiducek — APPROVED** [review]

**Comment by ChunxiAlexLuo** [comment]
> Holy moly! Removing PatternFlyTable is 2,400 lines of code??? RIP 🤣
> 
> This looks good to me, but since it's deferred for `2.4`, we should wait until that's open to merge it so CICD isnt' caught off guard.
> 
> /lgtm
> /hold

I'm fine with this

**Pull request merged**
